### PR TITLE
DROOLS-2270: [DMN Editor] Replace 'Return to DRG' button with Link

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2270

@jomarko Due to [this](https://issues.jboss.org/browse/DROOLS-1937) the name of the node shown in the "back link" in the UI is always "My name" however you can see from the Unit Test that it does use the value extracted from the ```HasName``` instance. I also stepped through the code in the browser and the ```HasName``` passed is identical to the node selected in the graph.

### CODE REVIEW

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionEditorViewImpl.java [hunk 1]
```diff
@@ -57,8 +59,8 @@
 
     private ExpressionEditorView.Presenter presenter;
 
-    @DataField("exitButton")
-    private Div exitButton;
+    @DataField("returnToDRG")
+    private Anchor returnToDRG;
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionEditorViewImpl.java]: I see the `org.jboss.errai.common.client.dom.Anchor` is deprecated. I tried to replace with `elemental2.dom.HTMLAnchorElement;` what was quite straightforward. Just replace `anchor.setTextContent()` with `anchor.textContent =`, however this is not testable. SO please consider this Deprecated class but easily testable vs. Non deprecated class but not so easily testable. Let me know your preference.

Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionEditorViewImpl.java]: Yes, I have "move to Elemental2" on my TODO list: https://issues.jboss.org/browse/DROOLS-1963

Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionEditorViewImpl.java]: You don't stop surprising me. Thanks.

**Comment by manstis** [comment]
@jomarko Test updated following your recommendation.

**Pull request merged**